source/documentation/disaster_recovery_guide/task/task_ExecuteFailover.adoc

```diff
@@ -10,17 +10,20 @@
 ** The `oVirt.disaster-recovery` package.
 ** The mapping file and required failover playbook.
 
-IMPORTANT: Sanlock must release all storage locks from the replicated storage domains before the failover process starts.
+[IMPORTANT]
+====
+Sanlock must release all storage locks from the replicated storage domains before the failover process starts.
 These locks should be released automatically approximately 80 seconds after the disaster occurs.
+====
 
 This example uses the `dr-rhv-failover.yml` playbook created earlier.
 
-*Executing a failover*:
+.Procedure
 
-Run the failover playbook with the following command:
+. Run the failover playbook with the following command:
```

Comment:
This and next sentences have "1." numbering both.

Reply:
Good catch! Fixed.